conclave-host/src/main/kotlin/com/r3/conclave/host/internal/gramine/GramineEnclaveHandle.kt

```diff
@@ -26,6 +31,10 @@ class GramineEnclaveHandle(
 
     companion object {
         private val logger = loggerFor<GramineEnclaveHandle>()
+        private const val GRAMINE_SECCOMP = "gramine-seccomp.json"
+
+        private val dockerImageTag =
+            getManifestAttribute(PluginUtils::class.java.classLoader, "Conclave-Build-Image-Tag")
```

Comment:
You don't need to pass in the classloader. Have `getManifestAttribute` use it directly.

Reply:
Yes, more or less.

Reply:
Sorry, I realize what you mean now. In Filipe's PR we have the attributes all in the same module. Will do the same, now I can actually merge, so things are easier